Apps/VisionOS/VisionOS/CommonViews/CommonLayoutView.swift

```diff
@@ -0,0 +1,65 @@
+import MarkdownUI
+import SwiftUI
+
+struct CommonLayoutView<ContentView: View>: View {
+    @ObservedObject var state = AppState.shared
+
+    @ViewBuilder var contentView: ContentView
+
+    var body: some View {
+        ZStack {
+            VStack(alignment: .leading, spacing: 8) {
+                HStack(alignment: .center) {
+                    TutorialsMenuView()
+
+                    Markdown {
+                        "## \(state.titleConfig.screenTitle)"
+                    }
+                    if state.titleConfig.showVisionProLogo {
+                        Image(systemName: "visionpro.fill")
+                            .font(.extraLargeTitle)
+                    }
+                    Spacer()
+
+                    Logo()
+                }
+                .frame(height: 50)
+                .clipped()
+                contentView
+                Spacer()
+            }
+            .markdownTheme(.tutorial)
+            .markdownCodeSyntaxHighlighter(.splash)
+
+            .padding([.top, .horizontal], 32)
+            .padding(.bottom, 16)
+            .textFieldStyle(.roundedBorder)
+
+            VStack {
+                Spacer()
+                ErrorToastView(error: $state.errorMessage)
+                SuccessToastView(message: $state.successMessage)
+            }
+        }
+        .buttonStyle(CustomButtonStyle())
+    }
+}
+
+struct CustomButtonStyle: PrimitiveButtonStyle {
+    typealias Body = Button
+    func makeBody(configuration: Configuration) -> some View {
+        Button(configuration)
+            .buttonBorderShape(.roundedRectangle(radius: 10))
+            .background(Color(rgba: 0x333333FF))
```

Comment:
Will this color be used in multiple parts of the app? If so, I think it's best to not have it hard-coded in a View. 

Perhaps use extensions so we could do: `Color.foo` to reference it?

Reply:
So far this is only used here. Agreed if at any point I needed to repeat it, it will be extracted out